Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -55,8 +55,7 @@ file that was distributed with this source code.
                         'code':     sonata_admin.field_description.associationadmin.code,
                         'objectId': sonata_admin.field_description.associationadmin.id(sonata_admin.value),
                         'uniqid':   sonata_admin.field_description.associationadmin.uniqid
-                        })
-                    )}}
+                        })) }}
```

Comment:
Could you tell me, how am I supposed to know it?
I did not read it in PSR-2 / PSR-1 or Symfony coding standards.
It's just to avoid to make any new mistake